### ISSUE TITLE
Lime API code 429 restriction

### ISSUE THREAD

**Issue by luyuliu** [ubahnverleih/WoBike#147]
Anyone met this error recently, maybe since last December?

**Comment by LilithWittmann** [comment]
Yes, it seems like that they introduced a relative low request limit :-/

**Comment by derwaldgeist** [comment]
The error came at the very first time for me. Never received an SMS.
